scss/base/_base.scss

```diff
@@ -1,34 +1,21 @@
-/* Resetting */
-*,
-*::before,
-*::after {
-  box-sizing: border-box;
-  margin: 0;
-  padding: 0;
-}
-
-/* Setting the value of 1rem */
 html {
-  /* 1rem = 10px */
+  // Setting the value of 1rem to 10px
   font-size: 62.5%;
-}
+  box-sizing: border-box;
 
-/* 20em = 320px */
-@media only screen and (max-width: 20em) {
-  html {
+  // screen-xsm = 20em = 320px
```

Comment:
- [ ] **[REQUIRED CHANGES]**

- Please, kindly remove unused code, it makes your codebase not clean.